### ISSUE TITLE
Crash in NNCoreWindowController

### ISSUE THREAD

**Issue by numist** [numist/Switch#32]
Exception Type: EXC_BAD_ACCESS
Code: KERN_INVALID_ADDRESS at 0x32

NNCoreWindowController.m line 252 -[NNCoreWindowController cellForWindow:]
NNCoreWindowController.m line 204 __40-[NNCoreWindowController setUpReactions]_block_invoke_2195

**Comment by numist** [comment]
Crash is coming from:

```
libobjc.A.dylib  _class_setInstancesHaveAssociatedObjects + 9
libobjc.A.dylib  _object_set_associative_reference + 443
```

Which is kinda scary.

**Comment by numist** [comment]
Unless this is a tools problem, this should be fixed with 216bc69e938edb722f27c068c576c9c3b062c5a2 and b128f186b51affadfbf3674a79170f800afe5701.
